linux_os/guide/services/dns/disabling_dns_server/package_bind_removed/rule.yml

```diff
@@ -1,6 +1,6 @@
 documentation_complete: true
 
-prodtype: rhcos4,ol7,ol8,rhel6,rhel7,rhel8,rhv4
+prodtype: rhcos4,ol7,ol8,rhel6,rhel7,rhel8,rhv4,fedora
```

Comment:
Nitpick, could you please sort it alphabetically?

Reply:
Sure, fixed.